stage0/src/fw_cfg.rs

```diff
@@ -225,26 +233,25 @@ impl FwCfg {
     }
 
     /// Reads the address for the intended location or the initial RAM disk.
-    pub fn read_initrd_address(&mut self) -> Result<VirtAddr, &'static str> {
+    pub fn read_initrd_address(&mut self) -> Result<PhysAddr, &'static str> {
         let mut initrd_addr: u32 = 0;
         self.write_selector(FwCfgItems::InitrdAddr as u16)?;
         self.read(&mut initrd_addr)?;
         // Since we use an identity mapping in the Stage0 firmware, we can interpret the physical
```

Comment:
This comment is now out of date, as we're not returning a VirtAddr.

Reply:
Thanks for catching it. Done.